### PR TITLE
added method doc

### PR DESCRIPTION
**Pull request by rajeshgheware**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def list_properties_file():
         return "Properties file not found."
     except Exception as e:
         return f"An error occurred: {e}"
+
+# http://localhost:5000/place/berlin
 @app.route('/place/<name>')
 def get_location_details(name):
     API_URL_GEOCODING = os.environ.get('API_URL_GEOCODING')
```
Comment (brainupgrade-in) [inline app.py]: Remove the extra line

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
Incorporate changes and re-request review

**Review by rajeshgheware — COMMENTED** [review]
incorporated changes

**Comment by rajeshgheware** [comment]
Done

**Review by brainupgrade-in — APPROVED** [review]
approving the changes

**Pull request merged**